euphonic/plot.py

```diff
@@ -171,11 +178,17 @@ def plot_1d(spectra: Spectrum1D | Spectrum1DCollection | Sequence[Spectrum1D] |
         # Check units are consistent
         for spectrum in spectra[1:]:
             if spectrum.x_data_unit != spectra[0].x_data_unit:
-                raise ValueError('Something went wrong: x data units are not '
-                                 'consistent between spectrum subplots.')
+                msg = (
+                    'Something went wrong: x data units are not '
```

Comment:
Something went wrong is kind of implied by the `raise`, but don't want to change messages this PR

Reply:
Hopefully will be spotted while reformatting error messages for #405